### PR TITLE
Feat/hng 69 blog post header

### LINKED ISSUE #69: [FEAT] External Dynamic Pages > Blog Post header
## Description
Develop a reusable Blog Post Header component. This component will be used at the top of individual blog post pages. The Blog Post Header should be visually appealing, responsive, and should include the following elements:

## Acceptance Criteria
- Implementation must be responsive
- All buttons should be functional
- Implementation must be pixel perfect

### Requirements	
The component should receive the following props:

- Title
- Subtitle
- Date
- Author name
- Author profile picture
- Tag
- Time of reading
- Blog Image (background image or main image)
- Excerpt

### Testing

- Ensure the component is responsive and renders correctly with all required elements
- Check that the component handles any length of texts (long titles, missing images) no weird layout shifts.

### Tasks

- [ ]  Develop the Blog Post Header component based on the provided requirements.
- [ ] Design should be responsive on all screens.
- [ ] Run tests to ensure the component works as expected.
- [ ] the component should receive props as described above under requirements.

# Link
[`FIGMA LINK`](https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=4-25687&m=dev).
### 
image

### PR DESCRIPTION
**Pull request by kingstech001**
# Description
Develop a reusable Blog Post Header component. This component will be used at the top of individual blog post pages. The Blog Post Header should be visually appealing, responsive, and should include the following elements:

**Closes #69**

# Changes proposed

## What were you told to do?
To develop a Blog Post Header component that receive the following props:

Title
Subtitle
Date
Author name
Author profile picture
Tag
Time of reading
Blog Image (background image or main image)
Excerpt
likes
comments

External Dynamic Pages > Blog Post header
## What did you do?
 Develop a Blog Post Header component based on the provided requirements.
 made the design responsive on all screens.
 Run a tests to ensure the component works as expected.
 the component should receive props as described above under requirements.

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **dev branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

**Comment by oreoluwa212** [comment]
```
[This branch has conflicts that must be resolved
 to resolve conflicts before continuing.
Conflicting files
package-lock.json
src/app/page.tsx]
```

**Review by oreoluwa212 — COMMENTED** [review]
The accepted folder structure for images is  `public > images > yourpagename `
Fix up!

Also, pull from upstream, accept only incoming for where you have conflicts. You are not meant to show anything on my page.tsx file. You have a header component, the blog page will use your comp when your PR is finally merged

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="./favicon.svg" type="image/svg+xml">
```
Comment (oreoluwa212) [inline html/index.html:5]: You must be joking!!!!!

index.html how???? Go and fix this, you're writing a next project

File: tailwind.config.ts [hunk 2]
```diff
@@ -18,6 +18,9 @@ const config = {
       },
     },
     extend: {
+      lineHeight: {
+        'custom-58': '58.09px',
```
Comment (oreoluwa212) [inline tailwind.config.ts:22]: Remove this! Follow what we have set up, the colors can be found on `localhost:3000/guides`  for you to see easily

Comment (oreoluwa212) [reply tailwind.config.ts:22]: Also, attach screenshot and video of your component 

Also, screenshots of lint and pnpm tests

File: html/index.html (hunk 1 above)
Comment (kingstech001) [reply html/index.html:5]: this is not coming from the component i worked on but the one i pulled from upstream

**Pull request closed without merging**